### PR TITLE
Fixes: Contact form submit button does not show up when screen resolution height <= 690px

### LINKED ISSUE #1340: Contact form submit button does not show up when screen resolution height <= 690px
**Describe the bug**
Contact form `submit button` does not show up when screen resolution height <= 690px. Need to account for fixed height footer, which is blocking the submit button when there is not enough vertical real-estate to display the entire contact form

**To Reproduce**
1. Go to https://dev.311-data.org/contact
2. Zoom into the page until the submit button disappears

**Expected behavior**
Should be able to scroll down to the submit button. Presently, contact form allows one to scroll down as far as the bottom of the message box

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser Chrome
 - Version 105.0.5195.102 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by edwinjue**
Added a grid container to bottom of contact form with a height equivalent to the footer to compensate for a fixed positioned footer that blocks visibility of portions of the contact form, namely `submit button`. Submit button now appears when scrolling all the way down to bottom when there is not enough screen real-estate as per attached screenshot (compare with attachment in #1340)

Alternative fixes:
Change `position: 'fixed'` to `position: 'sticky'` in [Footer](https://github.com/hackforla/311-data/blob/595319c0e176fcf41d60d0b8d1d10e1e95f8614d/client/components/Footer.jsx#L12) as per [stackoverflow](https://stackoverflow.com/questions/16950213/how-to-prevent-fixed-positioned-element-overlapping-others-elements-in-css) (see which browsers support [position: 'sticky'](https://caniuse.com/css-sticky))

Fixes #1340

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

**Review by nichhk — CHANGES_REQUESTED** [review]
I prefer changing the positioning of the footer, since this is an issue that could affect all of our pages (not just contact). Wdyt?

**Comment by edwinjue** [comment]
Thanks for the feedback Nich. Sure, that works for me as long as you are okay with `position: sticky` not being available on older browsers. Although we use MUI, we should keep in mind there are still a few older browsers where `position: sticky` isn't available where mui is still supported.

- [position: sticky browser compatibility](https://caniuse.com/css-sticky)
- [MUI browser compatibility](https://stackoverflow.com/questions/72390969/material-ui-browser-compatibility)

Additionally, it currently seems the other pages may be utilizing some bottom padding to mitigate the issue:
https://github.com/hackforla/311-data/blob/c0a2caf10d539b84a6fd25ee46e8e665480abf2e/client/components/main/About.jsx#L35

**Comment by edwinjue** [comment]
ok, in the latest PR, I restored my original contact form design and updated position in the footer component from `position: fixed` to `position: sticky`. I confirmed the document will scroll down to the submit button when screen resolution height <= 690px. the only other page I was able to verify is unaffected by the change is About page. the other pages on my local copy seem to want to load content from contentful which I do not have an API key for

File: client/components/contact/__tests__/__snapshots__/contactForm.test.js.snap [hunk 1]
```diff
@@ -1,299 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (nichhk) [inline client/components/contact/__tests__/__snapshots__/contactForm.test.js.snap:1]: Please revert this change from this PR

**Review by nichhk — CHANGES_REQUESTED** [review]
Looks good! One small comment.

Re: other pages compensating for the old `fixed` behavior: can you open an issue to investigate and fix those instances?

Re: browser compatibility: yeah, I took a look, and I don't think we need to worry about those :)

File: client/components/contact/__tests__/__snapshots__/contactForm.test.js.snap (hunk 1 above)
Comment (edwinjue) [reply client/components/contact/__tests__/__snapshots__/contactForm.test.js.snap]: ok, added back

**Review by nichhk — APPROVED** [review]

**Pull request merged**